app-frontend/src/app/components/common/search/search.js

```diff
@@ -7,7 +7,8 @@ const searchComponent = {
         autoFocus: '<',
         disabled: '<',
         placeholder: '@',
-        onSearch: '&'
+        onSearch: '&',
+        darkBg: '<?'
```

Comment:
I'd rather do this through a class on the component. eg: `<component class="component-dark"></component>`